packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -31,143 +32,142 @@ function fontFamilyFromSettings() {
-
-// 	// 		.cm-search-marker-selected {
-// 	// 			background: ${theme.selectedColor2};
-// 	// 			color: ${theme.color2} !important;
-// 	// 		}
-
-// 	// 		.cm-search-marker-scrollbar {
-// 	// 			background: ${theme.searchMarkerBackgroundColor};
-// 	// 			-moz-box-sizing: border-box;
-// 	// 			box-sizing: border-box;
-// 	// 			opacity: .5;
-// 	// 		}
-
-// 	// 		/* We need to use important to override theme specific values */
-// 	// 		.cm-error {
-// 	// 			color: inherit !important;
-// 	// 			background-color: inherit !important;
-// 	// 			border-bottom: 1px dotted #dc322f;
-// 	// 		}
-
-// 	// 		/* The default dark theme colors don't have enough contrast with the background */
-// 	// 		.cm-s-nord span.cm-comment {
-// 	// 			color: #9aa4b6 !important;
-// 	// 		}
-
-// 	// 		.cm-s-dracula span.cm-comment {
-// 	// 			color: #a1abc9 !important;
-// 	// 		}
-
-// 	// 		.cm-s-monokai span.cm-comment {
-// 	// 			color: #908b74 !important;
-// 	// 		}
-
-// 	// 		.cm-s-material-darker span.cm-comment {
-// 	// 			color: #878787 !important;
-// 	// 		}
-
-// 	// 		.cm-s-solarized.cm-s-dark span.cm-comment {
-// 	// 			color: #8ba1a7 !important;
-// 	// 		}
-
-// 	// 		/* MOBILE SPECIFIC */
-
-// 	// 		.CodeMirror .cm-scroller,
-// 	// 		.CodeMirror .cm-line {
-// 	// 			font-family: ${fontFamily};
-// 	// 			caret-color: ${caretColor};
-// 	// 		}
-
-// 	// 		${selectionColorCss}
-// 	// 	`);
-// 	// }, [themeId]);
-
-// 	return css;
-// }
+function useCss(themeId: number): string {
+	const [css, setCss] = useState('');
+
+	useEffect(() => {
+		const theme = themeStyle(themeId);
+
+		// Selection in dark mode is hard to see so make it brighter.
+		// https://discourse.joplinapp.org/t/dragging-in-dark-theme/12433/4?u=laurent
+		const selectionColorCss = theme.appearance === ThemeAppearance.Dark ?
+			`.CodeMirror-selected, div[class*=cm-selection] {
+				background: #6b6b6b !important;
+			}` : '';
+		const monospaceFonts = [];
+		// if (Setting.value('style.editor.monospaceFontFamily')) monospaceFonts.push(`"${Setting.value('style.editor.monospaceFontFamily')}"`);
+		monospaceFonts.push('monospace');
+
+		const fontSize = 15;
+		const fontFamily = fontFamilyFromSettings();
+
+		// Adjust the caret color for visibility when using a dark theme.
+		const caretColor = theme.appearance === ThemeAppearance.Dark ? 'white' : 'black';
+
+		setCss(`
+			/* These must be important to prevent the codemirror defaults from taking over*/
+			.CodeMirror {
+				font-family: ${fontFamily};
+				font-size: ${fontSize}px;
+				color: ${theme.color};
+				background-color: ${theme.backgroundColor};
+				/* position: absolute !important; */
+				-webkit-box-shadow: none !important; // Some themes add a box shadow for some reason
+			}
+
+			.CodeMirror-lines {
+				/* This is used to enable the scroll-past end behaviour. The same height should */
+				/* be applied to the viewer. */
+				/* Disabled on mobile */
+				/* padding-bottom: 400px !important; */
```

Comment:
Another change from the above CSS. Is it worth just commenting this out? Should it be removed entirely?